app/views/content/thank-you.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Thank you"
+description: |-
+  Thank you to candidates who have filled in a Get Into Teaching survey.
+date: "2021-11-01"
+image: "media/images/content/hero-images/0007.jpg"
+backlink: "../../"
+navigation: 40
```

Comment:
```suggestion
```

Reply:
Thanks @peteryates  - do we need a separate PR for noindexing? I'll raise a ticket